integration/auction_test.sh

```diff
@@ -21,37 +24,103 @@ CC_VERS=0
 num_rounds=3
 num_clients=10
 
+# Try function which returns the response string
+try_r() {
+    echo "$@" 
+    export RESPONSE=""
```

Comment:
do you need RESPONSE in sub-shells?  Otherwise i wouldn't export it, just set ..

Reply:
I tried it without exporting it and it wasn't visible outside the function

Reply:
hmm, unless you do it somewhere in a sub-shell, you do not have to export variables.

E.g.,
````
FOO=foo
foo() {
   FOO=bar
}
foo
echo $FOO
```
returns `bar`, not `foo` in bash